contracts/attesters/hydra-s1/HydraS1AccountboundAttester.sol

```diff
@@ -0,0 +1,286 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+pragma experimental ABIEncoderV2;
+
+import {IHydraS1AccountboundAttester} from './interfaces/IHydraS1AccountboundAttester.sol';
+import {HydraS1SimpleAttester} from './HydraS1SimpleAttester.sol';
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+
+// Core protocol Protocol imports
+import {Request, Attestation, Claim} from '../../core/libs/Structs.sol';
+import {IAttester} from '../../core/Attester.sol';
+
+// Imports related to HydraS1 Proving Scheme
+import {HydraS1Base, HydraS1Lib, HydraS1ProofData, HydraS1ProofInput, HydraS1Claim} from './base/HydraS1Base.sol';
+
+/**
+ * @title  Hydra-S1 Accountbound Attester
+ * @author Sismo
+ * @notice This attester is part of the family of the Hydra-S1 Attesters.
+ * Hydra-S1 attesters enable users to prove they have an account in a group in a privacy preserving way.
+ * The Hydra-S1 Simple Attester contract is inherited and holds the complex Hydra S1 verification logic.
+ * Request verification alongside proof verification is already implemented in the inherited HydraS1SimpleAttester, along with the buildAttestations logic.
+ * However, we override the buildAttestations function to encode the nullifier and its burn count in the user attestation.
+ * The _beforeRecordAttestations is also overrided to fit the Accountbound logic.
+ * We invite readers to refer to:
+ *    - https://hydra-s1.docs.sismo.io for a full guide through the Hydra-S1 ZK Attestations
+ *    - https://hydra-s1-circuits.docs.sismo.io for circuits, prover and verifiers of Hydra-S1
+
+ * This specific attester has the following characteristics:
+
+ * - Zero Knowledge
+ *   One cannot deduct from an attestation what source account was used to generate the underlying proof
+
+ * - Non Strict (scores)
+ *   If a user can generate an attestation of max value 100, they can also generate any attestation with value < 100.
+ *   This attester generate attestations of scores
+
+ * - Nullified
+ *   Each source account gets one nullifier per claim (i.e only one attestation per source account per claim)
+ *   For people used to semaphore/ tornado cash people:
+ *   nullifier = hash(sourceSecret, externalNullifier) <=> nullifierHash = hash(IdNullifier, externalNullifier)
+ 
+ * - Accountbound (with cooldown period)
```

Comment:
must say that each group has its own cooldown duration

must say that if cooldown duration = 0 => group does not allows account bound attestations

Reply:
must say that Accountbound is optionnal only, first line. so if they don't read the following they still know

Reply:
must say that Accountbound is optionnal only, first line. so if they don't read the following they still know